### PR TITLE
adding changelog entry for ansible upgrade

### PR DESCRIPTION
**Pull request by prem-fissionhq**
https://dimagi-dev.atlassian.net/browse/SAAS-12902
##### ENVIRONMENTS AFFECTED

@dannyroberts please review and let me know your suggestion

### CODE REVIEW

**Review by millerdev — COMMENTED** [review]
Can we do this automatically in the `init.sh` script just before running `pip-sync requirements.txt`? That is done in two places:

https://github.com/dimagi/commcare-cloud/blob/13515375e68d75192c1d1bcf89b431c0f5146b79/control/init.sh#L68
https://github.com/dimagi/commcare-cloud/blob/13515375e68d75192c1d1bcf89b431c0f5146b79/control/init.sh#L76

Ideally that script should be updated to check for the old version of ansible and uninstall it if it is installed.

A change log entry may make sense for communicating about how to upgrade ansible on developer workstations.

File: changelog/0049-upgrade-ansible.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  As part of our ongoing effort to keep CommCare HQ up to date with the latest tools and
+  libraries we have updated the Ansible version to use the ansible==4.2.0. During that
+  process we have test this approach on new servers and we have to uninstall the existing version 
+  and install the 4.2.0 
+
+update_steps: |
+  Run the following command to remove existing ansible:
+
+  ```bash
+  cchq <env> run-shell-command "pip uninstall ansible"
```
[the remark this answers is not in the document]
Comment (dannyroberts) [reply changelog/0049-upgrade-ansible.yml:20]: @prem-fissionhq I agree with Daniel here.

More generally, make sure to test exactly how the procedure will work, from the perspective of someone reading this the first time. So, if you revert back to the old version of ansible, and check that basic ansible commands work, and then follow the steps you lay out here, do you end up with a working ansible version of the new installation?

Ideally, try that both locally and on the control machine and make sure it works as you've described in each of those contexts

[the remark this answers is not in the document]
Comment (prem-fissionhq) [reply changelog/0049-upgrade-ansible.yml:20]: @dannyroberts sure. 
Thansk @millerdev

**Comment by prem-fissionhq** [comment]
@dannyroberts closing this PR to include this change in this PR: https://github.com/dimagi/commcare-cloud/pull/5326
cc: @millerdev

**Pull request closed without merging**